.kokoro/nightly/tutorials-samples.cfg

```diff
@@ -0,0 +1,33 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+          key: "TRAMPOLINE_IMAGE"
+          value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+          key: "JOB_TYPE"
+          value: "tutorials-samples"
+}
+
+# TODO: remove this after we've migrated all tests and scripts
+env_vars: {
+          key: "PROJECT_NUMBER"
+          value: "779844219229"
+}
+
+env_vars: {
+          key: "PROJECT_ID"
+          value: "java-docs-samples-testing"
+}
+
+env_vars: {
+          key: "GOOGLE_APPLICATION_CREDENTIALS"
+          value: "secret_manager/java-docs-samples-service-account"
+}
```

Comment:
In general we usually try to avoid putting secrets here and [instead put them in secret-manager](https://github.com/googleapis/java-retail/blob/d8e0c24668088ac1bb68470419b266003922aa08/.kokoro/build.sh#L87).

Reply:
This issue is still open.

Reply:
This is in secret manager. This just configures the env var for ADC